### ISSUE TITLE
Allow attemp_n3d in 3DS request

### ISSUE THREAD

**Issue by simas-ccore** [checkout/checkout-sdk-php#88]
Right now the SDK only supports enabling 3DS. Would it be possible to add **attempt_n3d** option to the 3DS constructor?

**Comment by rsaestrela** [comment]
Hi @simas-ccore. We've just updated the codebase and released version 2.0.0-beta3 that contains a new instantiation layer, compatibility with different API Keys, support for OAuth, custom logging & http client and some other interesting features.

The new codebase Payments client is 100% compatible with the API reference and it should be straightforward to implement your use case. Here's a glimpse on how you can request a 3ds/attempt_n3d payment:

```
$requestCardSource = new RequestCardSource();
$requestCardSource->name = "name"
$requestCardSource->number = "number"
$requestCardSource->expiry_year = 2026
$requestCardSource->expiry_month = 12
$requestCardSource->cvv = 123

$threeDsRequest = new ThreeDsRequest();
$threeDsRequest->enabled = true;
$threeDsRequest->attempt_n3d = true;
$threeDsRequest->eci = "eci"; // needed
$threeDsRequest->cryptogram = "cryptogram";  // needed
$threeDsRequest->xid = "";  // needed
$threeDsRequest->version = "2.0.1";

$paymentRequest = new PaymentRequest();
$paymentRequest->source = $requestCardSource;
$paymentRequest->three_ds = $threeDsRequest;

$paymentResponse = $api->getPaymentsClient()->requestPayment($paymentRequest);
``` 

To get started, please have a look on the project documentation (README). In the next days, we will upload [extensive docs](https://github.com/checkout/checkout-sdk-php#exception-handling) on how to use each operation. Feel free to create an issue if you have any questions or difficulties migrating to the new version.

**Comment by simas-ccore** [comment]
Thank you for the information. Good work!
